ParserXML/Parser.cs

```diff
@@ -1,46 +1,74 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
+using System.Runtime.Remoting.Services;
 using System.Xml.Linq;
 
-public class Contact
-{
-    public string Name { get; set; }
-    public Dictionary<string, string> Phones { get; set; }
-    public Address Address { get; set; }
-    public decimal? NetWorth { get; set; }
-
-    public Contact()
-    {
-        Phones = new Dictionary<string, string>();
-    }
-}
-
-public class Address
-{
-    public string Street1 { get; set; }
-    public string City { get; set; }
-    public string State { get; set; }
-    public string Postal { get; set; }
-}
-
 public class Program
 {
+    /// <summary>
+    /// New list
+    /// </summary>
     public static List<Contact> Contacts = new List<Contact>();
 
-    public static void Main(string[] args)
+    public static void Main()
     {
-        string filePath = "contacts.xml"; 
+        string filePath = "contacts.xml";
         ParseContacts(filePath);
 
         while (true)
         {
-            Console.WriteLine("\n1. Display All Contacts");
+            Console.WriteLine("1. Display Contacts");
+            Console.WriteLine("2. Search Contacts");
+            Console.WriteLine("3. Exit");
```

Comment:
You should add something like "Please choose the option by printing it's number."